scripts/check_style.sh

```diff
@@ -42,6 +49,12 @@ function preparedGrep
     return $?
 }
 
+function preparedGrepInIncludedDirectories
+{
+    grep -nIE -d skip "$1" ${INCLUDE_DIRECTORIES_JOINED}
+    return $?
```

Comment:
Why do you need the `return $?`, I believe it is sufficient to just perform the grep.
I.e.
```sh
function preparedGrepInIncludedDirectories
{
    grep -nIE -d skip "$1" ${INCLUDE_DIRECTORIES_JOINED}
}
```
And maybe use it directly in `FORMATERROR` instead of a helper function.

Reply:
No idea, I just copied `preparedGrep`. In any case, this is gonna get removed as soon as `using namespace std;` is purged from the codebase, so I'd rather not invest more time into refactoring something that will just get deleted anyway in a couple of weeks.